package.json

```diff
@@ -45,6 +45,7 @@
     "@radix-ui/react-slot": "^1.1.0",
     "date-fns": "^4.1.0",
     "katex": "^0.16.11",
+    "lodash.debounce": "^4.0.8",
```

Comment:
Curious why you need debounce? Wouldn't a setTimeout/clearTimeout be enough for this?